src/main/java/com/shilymily/bms/service/CompanyService.java

```diff
@@ -0,0 +1,37 @@
+package com.shilymily.bms.service;
+
+import com.shilymily.bms.entity.Company;
+import com.shilymily.bms.repository.CompanyRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by hmaskai on 9/8/18.
+ */
+@Service
+public class CompanyService {
+
+    @Autowired
+    private CompanyRepository companyRepository;
+
+    public void addOrUpdateCompany(Company company){
+        companyRepository.save(company);
+    }
+
+    public void removeCompany(String name){
+        companyRepository.deleteByName(name);
```

Comment:
What happens when these actions fail? for eg: if the company doesn't exists in the database, what error do we propagate to the user? 
Or is it going to be like a dropdown option thingy?
Irrespective, I would suggest to throw user facing exceptions.

Reply:
Yea i was thinking of a good way of doing it....will add it in my next commit